safe/utilities/keyword_io.py

```diff
@@ -457,35 +463,127 @@ def _keyword_to_row(self, keyword, value, wrap_slash=False):
+                        },
+                        'active': False
+
+                    }
+                },
+                'population': {
+                    'ina_population_flood_hazard_classification': {
+                        'classes': {
+                            'low': [1, 2.5],
+                            'medium': [2.5, 4.5],
+                            'high': [4.5, 6]
+                        },
+                        'active': False
+                    },
+                    'ina_population_flood_hazard_4_class_classification':
+                    {
+                        'classes': {
+                            'low': [1, 2.5],
+                            'medium': [2.5, 4],
+                            'high': [4, 6],
+                            'very_high': [6, 8]
+                        },
+                        'active': True
+                    }
+                },
+            },
 
         Each value is a list with exactly two element [a, b], where a <= b.
 
-        :param keyword_value: Value of the keyword to be rendered. This must
-            be a string representation of a dict, or a dict.
-        :type keyword_value: basestring, dict
+        :param thresholds_keyword: Value of the keyword to be rendered. This
+            must be a string representation of a dict, or a dict.
+        :type thresholds_keyword: basestring, dict
 
         :returns: A table to be added into a cell in the keywords table.
         :rtype: safe.messaging.items.table
         """
-        if isinstance(keyword_value, basestring):
-            keyword_value = literal_eval(keyword_value)
+        if isinstance(thresholds_keyword, basestring):
+            thresholds_keyword = literal_eval(thresholds_keyword)
+
+        table = m.Table(style_class='table table-condensed table-striped')
+
+        i = 0
+        for exposure_key, classifications in thresholds_keyword.items():
+            i += 1
+            exposure = definition(exposure_key)
+            exposure_row = m.Row()
+            exposure_row.add(m.Cell(m.ImportantText('Exposure')))
+            exposure_row.add(m.Cell(m.Text(exposure['name'])))
+            exposure_row.add(m.Cell(''))
+            table.add(exposure_row)
+
+            active_classification = None
+            classification_row = m.Row()
+            classification_row.add(m.Cell(m.ImportantText('Classification')))
+            for classification, value in classifications.items():
+                if value.get('active'):
+                    active_classification = definition(classification)
+                    classification_row.add(
+                        m.Cell(active_classification['name']))
+                    classification_row.add(m.Cell(tr('')))
+                    break
+
+            if not active_classification:
+                classification_row.add(m.Cell(tr('No classifications set.')))
+                classification_row.add(m.Cell(tr('')))
```

Comment:
I think @ismailsunni is cut and pasting my code probably - I think I used tr('') when I had some unicode issues with the generated html....